### ISSUE TITLE
replacing onboard Intel Wi-Fi 6 802.11ax possible?

### ISSUE THREAD

**Issue by tristan-k** [SchmockLord/Hackintosh-Intel-i9-10900k-Gigabyte-Z490-Vision-D#11]
Is it possible to replace the onboard Intel Wi-Fi 6 802.11ax with a macOS compatible m.2 like the BCM94350ZAE?

**Comment by SchmockLord** [comment]
No. I have tried that yesterday. It didn't work. Then I found out that this m.2 slot is only for so-called CNVi Wireless cards. Only Intel makes them and they share their work with the CPU/RAM directly.

**Comment by tristan-k** [comment]
Thanks. I wonder if there is adapter or something else.

**Comment by SchmockLord** [comment]
No there is no adapter to get a non-CNVi Wifi card working in the CNVi slot.
